### PR TITLE
Fix key order in orderByDescAsc

### PR DESCRIPTION
**Pull request by jsantos17**

### CODE REVIEW

File: it/src/main/resources/tests/orderByDescAsc.test [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
     "backends": {
-        "couchbase":         "pending",
-        "marklogic_json":    "pending",
-        "marklogic_xml":     "pending",
-        "mongodb_3_2":       "pending",
-        "mongodb_3_4":       "pending",
-        "mongodb_read_only": "pending",
-        "spark_local":       "pending",
-        "spark_hdfs":        "pending"
+        "mimir":       "pending"
```
Comment (alissapajer) [inline it/src/main/resources/tests/orderByDescAsc.test]: `mimir` should pass if you use `ignoreFieldOrder`. `mimir` doesn't guarantee the results will have field order the same as in the query.

I believe that `marklogic_json` and `couchbase` also do not guarantee field order in the results, but @wemrysi and @drostron (or whoever knows) should confirm.

Comment (drostron) [reply it/src/main/resources/tests/orderByDescAsc.test]: Correct, `couchbase` does not guarantee field order.

Comment (jsantos17) [reply it/src/main/resources/tests/orderByDescAsc.test]: Marklogic does guarantee field order.  I'll update `couchbase` and `mimir` to `ignoreFieldOrder`. Thanks for all the input!

Comment (wemrysi) [reply it/src/main/resources/tests/orderByDescAsc.test]: @jsantos17 It does for `XML`, but I didn't think it did for `JSON`.

Comment (jsantos17) [reply it/src/main/resources/tests/orderByDescAsc.test]: Oh right. I might've extrapolated from `XML`. I'll set `marklogic_json` to `ignoreFieldOrder` too.

**Comment by jsantos17** [comment]
@alissapajer done! :)

**Pull request merged**